frontend/turbo.json

```diff
@@ -77,6 +77,14 @@
             "persistent": true,
             "dependsOn": ["^wait"]
         },
+        "android:release": {
+            "outputs": ["lib/**", "build/**"],
+            "dependsOn": [
+                "^android:release",
```

Comment:
Could this just depend on `build` (or maybe `^build`?), then you wouldn't need to add the new steps to each packag.json other than the app one.